django_inertia/assertions.py

```diff
@@ -1,11 +1,11 @@
 def assert_component(response, component):
     assert (
-        response.context["page"]["component"] == component
-    ), f"Asserted {component}, got {response.context['page']['component']}"
+        response.context["__page"]["component"] == component
+    ), f"Asserted {component}, got {response.context['__page']['component']}"
 
 
 def assert_props(response, key, value=None):
-    props = response.context["page"]["props"]
+    props = response.context["__page"]["props"]
```

Comment:
Same here, the value should not be hard-coded

Reply:
Resolved